arangod/Aql/AqlValue.cpp

```diff
@@ -552,6 +553,50 @@ AqlValue AqlValue::get(CollectionNameResolver const& resolver,
   return AqlValue(AqlValueHintNull());
 }
 
+/// @brief get the (object) element by name
+AqlValue AqlValue::get(CollectionNameResolver const& resolver,
+                       arangodb::velocypack::StringRef const& name, bool& mustDestroy, bool doCopy) const {
```

Comment:
can we perhaps typedef the `lib/Basics/StringRef.h` class to the velocypack one ?

Reply:
good idea. will do that!